src/ir/module-utils.cpp

```diff
@@ -33,6 +33,11 @@ struct Counts : public InsertOrderedMap<HeapType, size_t> {
       note(ht);
     }
   }
+  void include(HeapType type) {
```

Comment:
I had to read the body 3 times before I saw the difference is in the lack of `++`, so I think perhaps a comment could help here. "include" does not explain enough by itself, at least for me, the difference with "note".

Or, how about `note => count` and `include => ensure`?

Reply:
Unfortunately `count` is already the name of the method we use to determine whether a type is already present in the `Counts`, since it inherits from a map type.

I'll at least add a comment about the difference for now.